### PR TITLE
Core folder path is resolved with cross platform compatibility

### PR DESCRIPTION
**Pull request by Gusinuhe**
### Description

**Fixes**: Core folder used to work only for Windows, this PR adds cross-OS compatibility.

### Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: Editor/PackageManager/PrefabsValidator.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ private static void OnPostProcess(object sender, DependencyManager.DependenciesE
         private static void ReimportCreatorPrefabs()
         {
             string filter = "t:Prefab";
-            string[] prefabsGUIDs = AssetDatabase.FindAssets(filter, new[] {"Assets/Innoactive"});
+            string[] prefabsGUIDs = AssetDatabase.FindAssets(filter, new[] { EditorUtils.GetCoreFolder() });
```
Comment (tomwim) [inline Editor/PackageManager/PrefabsValidator.cs:26]: Is Core Folder "Assets/Innoactive" or "Assets/Innoactive/Core"?

Comment (Gusinuhe) [reply Editor/PackageManager/PrefabsValidator.cs:26]: It actually is `Assets\Innoactive\Creator\Core` :scream:
No prefab will be updated as this is right now, thanks for pointing it out!

File: Editor/PackageManager/PrefabsValidator.cs [hunk 2]
```diff
@@ -23,7 +24,9 @@ private static void OnPostProcess(object sender, DependencyManager.DependenciesE
         private static void ReimportCreatorPrefabs()
         {
             string filter = "t:Prefab";
-            string[] prefabsGUIDs = AssetDatabase.FindAssets(filter, new[] {"Assets/Innoactive"});
+            string[] prefabsGUIDs = AssetDatabase.FindAssets(filter, new[] { GetInnoactiveFolder() });
+
+            Debug.LogWarning(GetInnoactiveFolder());
```
Comment (tomwim) [inline Editor/PackageManager/PrefabsValidator.cs:29]: Guess this has to go

Comment (Gusinuhe) [reply Editor/PackageManager/PrefabsValidator.cs]: Whoops!

**Pull request merged**